packages/patch/patch.3.0.0~alpha1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "Patch library purely in OCaml"
+description: """\
+This is a library which parses unified diff and git diff output, and can
+apply a patch in memory."""
+maintainer: "Hannes Mehnert <hannes@mehnert.org>"
+authors: "Hannes Mehnert <hannes@mehnert.org>"
```

Comment:
please add yourself as author (+maintainer)

Reply:
done

Reply:
ported upstream in https://github.com/hannesm/patch/commit/21f062e5a68b374be35b852d3d6d264e49adc478